rusk-wallet/src/bin/command/tests/utils.rs

```diff
@@ -0,0 +1,324 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use std::collections::{HashMap, HashSet};
+use std::net::TcpStream;
+use std::time::Duration;
+
+use rusk_wallet::GraphQL;
+use serde::Deserialize;
+use tempfile::{tempdir, TempDir};
+use tracing_subscriber::EnvFilter;
+use url::Url;
+
+use super::*;
+use crate::command::history::TransactionDirection;
+use crate::settings::{LogLevel, Logging};
+use crate::{connect, status, LogFormat};
+
+#[derive(Default)]
+struct FakePrompter {
+    text_answer: String,
+}
+
+impl Prompt for FakePrompter {
+    fn create_new_password(
+        &self,
+    ) -> anyhow::Result<String, inquire::InquireError> {
+        Ok("password".to_string())
+    }
+
+    fn prompt_text(&self, _msg: &str) -> inquire::error::InquireResult<String> {
+        return Ok(self.text_answer.clone());
+    }
+}
+
+#[derive(Debug, PartialEq)]
+pub struct StrippedTxHistoryItem {
+    pub direction: TransactionDirection,
+    pub amount: f64,
+    pub fee: u64,
+}
+
+impl Into<StrippedTxHistoryItem> for TransactionHistory {
+    fn into(self) -> StrippedTxHistoryItem {
+        StrippedTxHistoryItem {
+            direction: self.direction,
+            amount: self.amount,
+            fee: self.fee,
+        }
+    }
+}
+
+pub fn configure_logger() {
+    let directive =
+        std::env::var("RUST_LOG").unwrap_or_else(|_| "info".to_string());
+
+    let filter = EnvFilter::new(directive);
+    let _ = tracing_subscriber::fmt().with_env_filter(filter).try_init();
+}
+const STATE_ADDRESS: &str = "http://127.0.0.1:8080";
+const PROVER_ADDRESS: &str = "http://127.0.0.1:8081";
```

Comment:
If you get this from env, you can inject the right one from `run-rusk-wallet-tests.sh`